Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,184 @@
+name: pradeo-mtd
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: metadata
+  - name: initiator
+  - name: permission
+  - name: detection
+  - name: correlation
+  - name: compliance
+stages:
+  metadata:
+    actions:
+      - set:
+          pradeo.metadata.id: '{{json_event.message.id}}'
+          pradeo.metadata.creationDate: '{{json_event.message.creationDate}}'
+          pradeo.metadata.source: '{{json_event.message.source}}'
+          pradeo.metadata.category: '{{json_event.message.category}}'
+          pradeo.metadata.type: '{{json_event.message.type}}'
+  initiator:
+    actions:
+      - filter: '{{json_event.message.source == "device" and json_event.message.device != null}}'
+        set:
+          pradeo.initiator.id: '{{json_event.message.device.id}}'
+          pradeo.initiator.serialNumber: '{{json_event.message.device.serialNumber}}'
+          pradeo.initiator.name: '{{json_event.message.device.name}}'
+          pradeo.initiator.email: '{{json_event.message.device.email}}'
```

[`user.email`](https://www.elastic.co/guide/en/ecs/8.11/ecs-user.html#field-user-email) is more suitable.
```suggestion
          user.email: '{{json_event.message.device.email}}'
```